### ISSUE TITLE
Move nextIcon mixin to a shiny new n-icon repo

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/next-sass-setup#89]
n-icon may also provide:

- A base `.n-icon` class
- Size variations for epic consistency `.n-icon--marketing-department-large`
- Colour variations because we should always stick to the colour palette `.n-icon--navajowhite`
- Class for each icon from next `.n-icon--right-arrow-with-a-ring-around-it`

And see where this may overlap with o-ft-icons.

**Comment by i-like-robots** [comment]
cc @wheresrhys @keirog

**Comment by wheresrhys** [comment]
alo see https://github.com/Financial-Times/o-ft-icons/issues/72

**Comment by keirog** [comment]
Yesss. One for bug challenge/duty/safari/experience?

**Comment by matthew-andrews** [comment]
Why do we need a new icon repo?  Can we not merge what we need into `o-icons`?

**Comment by matthew-andrews** [comment]
[ i mean … can this not be just a new major version of o-icons? ]

**Comment by wheresrhys** [comment]
Next (and I think the web dev community at large) is resolutely against icon fonts, but a feature of o-icons only really achievable with icon fonts is complete customisability of colours when using the build service (i.e. class defines the icon to use, but end user can define the colour as they wish). Also I think next's icon set is still evolving and we don't really want to clutter up the o-icons icon-set. 

At whichever point we feel next's iconography is settled enough I think we should 
- tackle the question of how important full colour customisability is, 
- pick the best implementation available at the time
- clear out all the old icons (redesigning to match the next look and feel when next hasn't already got an icon for that use case)

So to sum up, yes in the longer term (revisit in Q2?), but for now I think we need a separate component we can use to play around with stuff without affecting club origami.

**Comment by wheresrhys** [comment]
Also I tend towards thinking icons should be in the html rather than set with CSS. How does this fit with origami?

**Comment by matthew-andrews** [comment]
I think we all agreed the next version of o-icons should definitely ditch the fonts.

Origami copes with evolution (minor versions), we decided with Dan a while back to merge Origami and Next designs … we're a year into Next now … it's time to merge.

**Comment by matthew-andrews** [comment]
Also, like with colours, we can always extend the base icon set in next-sass-setup

**Comment by wheresrhys** [comment]
I don't think it's quite that simple, but probably one for a face to face talk. Origami meeting on thursday? @AlbertoElias @alicebartlett 

On the design side I would expect us to have a coherent, reasonably broad set of icons to hydrate o-icons@3. We shouldn't release it with a mish-mash of old and new icon styles - confusing for devs and bad for brand consistency = bad origami

**Comment by AlbertoElias** [comment]
It's a major release, we can start deprecating old icons in the current release and have them removed in the new release. And I'm definitely in favour of moving away from fonts if we have an implementation that works for everyone. We can definitely talk about this on Thursday :)

**Comment by i-like-robots** [comment]
I think this has probably been usurped by https://github.com/Financial-Times/o-ft-icons/pull/100 so I''m going to close it.
